RetrospectiveExtension.Frontend/components/feedbackColumn.tsx

```diff
@@ -300,3 +302,7 @@ export default class FeedbackColumn extends React.Component<FeedbackColumnProps,
     return this.renderFeedbackColumn();
   }
 }
+
+export { FeedbackColumn as FeedbackColumnStatic}; // static functions are not preserved with `withAITracking`,
```

Comment:
can we move static methods to a utility class and not export the same class in a different approach?

Reply:
makes sense